packages/cli/test/unit/commands/db/revert.test.ts

```diff
@@ -74,6 +74,45 @@ test("don't revert the last migration if it had no down migration", async () =>
 	expect(dataSource.destroy).not.toHaveBeenCalled();
 });
 
+test('print migration name in error message is the migration has a name', async () => {
+	//
+	// ARRANGE
+	//
+	class TestMigration implements IrreversibleMigration {
+		name = 'Test Migration';
+
+		async up() {}
+	}
+
+	const connectionOptions = DbConfig.getConnectionOptions();
+	const migrations = [TestMigration];
+	// @ts-expect-error property is readonly
+	connectionOptions.migrations = migrations;
+	const dataSource = mock<DataSource>();
+	// @ts-expect-error property is readonly, and I can't pass them the `mock`
+	// because `mock` will mock the down method and thus defeat the purpose
+	// of this test, because the tested code will assume that the migration has a
+	// down method.
+	dataSource.migrations = migrations.map((M) => new M());
+
+	//
+	// ACT
+	//
+	await main(connectionOptions, logger, function () {
+		return dataSource;
+	} as never);
+
+	//
+	// ASSERT
+	//
+	expect(logger.error).toHaveBeenCalledTimes(1);
```

Comment:
Is this redundant because of the next assertion?

Reply:
I want to make sure that only one message is printed, e.g. the logger is only called once. The next line just asserts that the logger was called with that param at all, so it could have been called multiple times as well.